### ISSUE TITLE
SourceRecordReader implementations should use milliseconds to create their records

### ISSUE THREAD

**Issue by ahawtho** [redis-field-engineering/redis-kafka-connect#15]
Kafka record timestamps are defined in milliseconds, but the [`StreamSourceRecordReader` class](https://github.com/redis-field-engineering/redis-kafka-connect/blob/master/src/main/java/com/redis/kafka/connect/source/StreamSourceRecordReader.java#L94
) and the [`KeySourceRecordReader` class](https://github.com/redis-field-engineering/redis-kafka-connect/blob/master/src/main/java/com/redis/kafka/connect/source/KeySourceRecordReader.java#L102) use the following to create their records:

```
		return new SourceRecord(..., Instant.now().getEpochSecond());
```

These should be switched to `Instant.now().toEpochMilli()`.

The effect is that when the source connector appends a message to a topic, if the topic has `retention.ms` set to a reasonable value, the messages are immediately deleted from the topic.

This can be worked around by setting the topic configuration value:
```
message.timestamp.type: LogAppendTime
```

**Comment by ahawtho** [comment]
Thanks for the quick turnaround. Any idea when will this make a release?

**Comment by jruaux** [comment]
Thank you for your contribution. This is included in version 7.2.

**Comment by jruaux** [comment]
🎉 This issue has been resolved in `v0.7.5` ([Release Notes](https://github.com/redis-field-engineering/redis-kafka-connect/releases/tag/v0.7.5))
